src/services/walkthroughServices.js

```diff
@@ -14,12 +14,14 @@ class NodeMessagingAppEnmassSiTransform {
   }
 
   transform(siInfo) {
-    const enmassCreds = {
+    const placeholderParameters = {
       MESSAGING_SERVICE_USER: 'username',
-      MESSAGING_SERVICE_PASSWORD: 'password'
+      MESSAGING_SERVICE_PASSWORD: 'password',
+      MESSAGING_SERVICE_HOST: 'messaging.enmasse-eval.svc',
```

Comment:
Yep, all these will need to be retrieved from EnMasse as they're all namespace specific. We can build this up on the client side if we want though as we know the name of the namespace (since we create it) and we know the service will be called messaging. So basically replace `enmasse-eval` with the known namespace name.

Reply:
yup, I was waiting for that to be ready, hence `placeholderParams` as a name

Reply:
I've added the url now as well: https://github.com/integr8ly/tutorial-web-app/pull/60/files#diff-ed7417901bdafe31ec92bf19d0ea8a42R53